src/Microsoft.Diagnostics.Monitoring.RestServer/OutputStreamResult.cs

```diff
@@ -20,11 +17,23 @@ internal sealed class OutputStreamResult : ActionResult
         private readonly string _contentType;
         private readonly string _fileDownloadName;
 
+        public OutputStreamResult(Func<CancellationToken, Task<Stream>> action, string contentType, string fileDownloadName = null)
+        {
+            _contentType = contentType;
+            _fileDownloadName = fileDownloadName;
+            _action = async (stream, token) =>
+            {
+                using var sourceStream = await action(token);
+
+                await sourceStream.CopyToAsync(stream, 0x1000, token);
```

Comment:
Note I recently updated similiar code to reflect the asp.net FileStreamResult, which is 0x10000 iirc.

Reply:
Fixed